### PR TITLE
SLIDER IS HERE

### PR DESCRIPTION
**Pull request by salarali**
BEWARE!!

### CODE REVIEW

File: src/js/popup.js [hunk 1]
```diff
@@ -839,6 +839,22 @@ var onHideTooltip = function() {
 
 /******************************************************************************/
 
+var changeGlobalSlider = function(event) {
+    console.log(event.target.value);
```
Comment (sangyounkim) [inline src/js/popup.js:843]: fix it @Zozos

Comment (salarali) [inline src/js/popup.js:843]: Fix it yourself @sangyounkim

Comment (sangyounkim) [inline src/js/popup.js:843]: you are fired

Comment (salarali) [inline src/js/popup.js:843]: Its okay. I hated working here anyway.

**Pull request merged**